.github/actions/deploy/action.yml

```diff
@@ -125,7 +125,7 @@ runs:
 
     - name: Run data migrations
       shell: bash
-      if: inputs.environment == 'staging' || inputs.environment == 'productiondata' || inputs.environment == 'production'
+      if: inputs.environment in ['staging', 'sandbox', 'productiondata', 'production']
```

Comment:
This syntax doesn't work in github actions: https://github.com/DFE-Digital/register-trainee-teachers/actions/runs/7784096429/job/21224021287?pr=4000
You may want [contains](https://docs.github.com/en/actions/learn-github-actions/expressions#contains) ?

Reply:
Fixed!